frontend/src/app/components/homescreen/blocks/new-features.component.ts

```diff
@@ -37,16 +37,18 @@ export const homescreenNewFeaturesBlockSelector = 'homescreen-new-features-block
 
 @Component({
   template: `
-    <p class="widget-box--additional-info">
-      {{ text.descriptionNewFeatures }}
-    </p>
-    <div class="widget-box--description">
-      <p [innerHtml]="currentNewFeatureHtml"></p>
-      <img [src]="new_features_image"/>
-      <a class="widget-box--teaser-image"></a>
-    </div>
+    <div class="op-new-feature">
+      <p class="widget-box--additional-info">
+        {{ text.descriptionNewFeatures }}
+      </p>
+      <div class="widget-box--description">
+        <p [innerHtml]="currentNewFeatureHtml"></p>
+        <img [src]="new_features_image"/>
+        <a class="widget-box--teaser-image"></a>
```

Comment:
In the sass you are selecting this class, even though you are working with the `op-new-feature` block. A more BEM approach to this would be to also make this component an element of `op-new-feature`. For example:

```suggestion
        <a class="widget-box--teaser-image op-new-feature--teaser-image"></a>
```

That way in the sass you can just select

```
.op-new-feature
  &--teaser-image
```

No need to write all the unrelated classes into the sass then